src/Microsoft.Health.Fhir.Shared.Api/Features/ContentTypes/ContentTypeService.cs

```diff
@@ -82,23 +83,30 @@ public async Task CheckRequestedContentTypeAsync(HttpContext httpContext)
                     }
                 }
             }
+
+            string prettyParameterValue = GetParameterValueFromQueryString(httpContext, KnownQueryParameterNames.Pretty);
+
+            if (prettyParameterValue != null && !bool.TryParse(prettyParameterValue, out _))
+            {
+                throw new BadRequestException(Api.Resources.UnsupportedPrettyParameter);
```

Comment:
nit: can we use `UnsupportedParameterValue` instead of adding a new one?

Reply:
I ended up changing this resource to better fit the error code that I'm returning in fccf656. Otherwise, I think you're right that it would have been nice to have a single message for both _pretty and _format.